src/js/tools/metroconfig.ts

```diff
@@ -146,6 +158,29 @@ function withSentryDebugId(config: MetroConfig): MetroConfig {
   };
 }
 
+/**
+ *
+ */
+export function excludeSentryWebReplay(config: MetroConfig, includeWebReplay: boolean | undefined): MetroConfig {
```

Comment:
Let's rename `excludeSentryWebReplay` to `includeSentryWebReplay` to match the flag name `includeWebReplay`.

To make it easier to read

`excludeSentryWebReplay(config, true)` -> web replay is included (but looks like exclude true)

vs

`includeSentryWebReplay(config, true)` -> web replay is included